Sponsor Update

Jacob-Mellichamp opened the pull request:
View the page at https://sponsor.dev.cuctf.io

Main take aways:
  - Added a link to the CUCTF Sponsor Packet on sponsors.html
  - Reformatted sponsors.html
  - Added CUCTF 2019 sponsors to the bottom of CUCTF_2019.html
  - Minor changes to the formatting of CUCTF_2019.html
  - Added logos images and packet pdf to the public folder. 

Let me know if this is ready for a merge or if additional work is needed

Jacob-Mellichamp committed:
Testing Look

Jacob-Mellichamp committed:
attached links to images

Jacob-Mellichamp committed:
sort rows

Jacob-Mellichamp committed:
center sponsor images

Jacob-Mellichamp committed:
Black backgrounds for images, positioned correctly

Jacob-Mellichamp committed:
Added more insight to CUCTF 2019

Jacob-Mellichamp committed:
Sponsor Page Includes Packet download and Benefits Image

Jacob-Mellichamp committed:
Removed Picture from sponsor page, reformatted page

lilyinstarlight reviewed (changes requested):
I'm putting this here mostly as a few notes for changing and merge notes but I'll provide a more specific review later. The merge notes below are for me or whoever is merging and the more specific review later will be made for the changes below. Adding last year's sponsors and the packet though is a much appreciated change!

### Changes

- I'm pretty sure at least Ally's logo and possibly others need to be against a white background per their brand policy. I can double check this later and provide a list of requirements and suggestions.
- Sponsor images need `alt` attribute filled out
- Move `style` attribute on `p` tag to either id or class style in stylesheet


### Merge notes

- There are a lot of errant spaces (only affects the code readability but not presentation) so I'll probably get rid of those before merge
- Remove .DS_Store files before merge
- Rebase on dev before merge

houstdav000 commented:
Just a recommendation, but I'd suggest creating a global .gitignore_global file to prevent adding .DS_Store files like [mine](https://github.com/houstdav000/dotfiles/blob/master/.gitignore_global)

Jacob-Mellichamp committed:
Implemented Changes suggested by foster

Jacob-Mellichamp committed:
All Level 2 sponsors Have white backgrounds

Jacob-Mellichamp commented:
## Pushed Changes to the branch:
  - Removed all .DS_Store files
  - Made Level 2 sponsors have white background
  - moved `<p style="">` to css class
  - added alt text to sponsor images
  - removed CUCTF Sponsor Benefits image from `/public/2020`
